docs/basics.txt

```diff
@@ -1,25 +1,21 @@
 [[basics]]
-JanusGraph Basics
-=================
+= JanusGraph Basics
 
 [[configuration]]
-Configuration
--------------
+== Configuration
 
-A JanusGraph graph database cluster consists of one or multiple JanusGraph instances. To open a JanusGraph instance a configuration has to be provided which specifies how JanusGraph should be set up.
+A JanusGraph server cluster consists of one or multiple JanusGraph server instances. To open a JanusGraph instance, a configuration has to be provided which specifies how JanusGraph should be set up.
```

Comment:
Will the use of term 'server' cluster confuse with the real Gremlin JanusGraph server? Implying that there are always Gremlin server involved (the so-called remote server mode) and excluding other deployment modes?